### ISSUE TITLE
CnnInYourOwnData：after call Train() ，in Test() function will interrupt

### ISSUE THREAD

**Issue by zj2050** [SciSharp/SciSharp-Stack-Examples#61]
In vs2019 debug mode，after CnnInYourOwnData  call Train() and running finished，it  can generated all ckpt files
but then call Test() function ：  (loss_test, accuracy_test) = sess.run((loss, accuracy), (x, x_test), (y, y_test)); will auto interrupted without any information after some time;

if deleted next two lines 
            Test();
            TestDataOutput();

and direct call  function
             FreezeModel();

        public override string FreezeModel()
        {
            return tf.train.freeze_graph(Config.Name + @"\\MODEL", @".\\CNN_Best", new[]
            {
                "Input/X", 
                "Input/Y", 
                "Train/Prediction/predictions",
                "Train/Prediction/prob"
            });
        }

will generate pb file, but the pb file is more less ckpt files，ckpt file more than 786M， and pb file is only 262M，using netron open pb and ckpt it show very defferent

![bbcb-1b55aad29374b9e1189c6f5003e79b8c](https://user-images.githubusercontent.com/4464947/143813354-0eb7ae3f-437e-45ea-ada8-2d3e9bd75f51.jpg)

thanks
